resources/views/home.blade.php

```diff
@@ -24,7 +24,13 @@ function setColorMode(dark) {
     <main class="container-md my-5">
         <div class="row">
             <div class="col-md-6 offset-md-3">
-                {{ html()->form('GET', '/pdf')->attribute('accept-charset', 'UTF-8')->open() }}
+                @php
```

Comment:
mind reverting this change, since it's not needed now?

Reply:
Yep. Thanks for catching!